--- origsrc/gmanedit-0.4.1/Makefile.am	2008-03-03 02:28:14.000000000 -0600
+++ src/gmanedit-0.4.1/Makefile.am	2008-04-15 01:33:38.046875000 -0500
@@ -14,8 +14,6 @@
 desktopdir = $(datadir)/applications
 desktop_DATA = $(DESKTOP_FILES)
 
-ACLOCAL_AMFLAGS = -I m4
-
 EXTRA_DIST = config.rpath $(DESKTOP_FILES) gmanedit.1
 
 uninstall-local:
--- origsrc/gmanedit-0.4.1/configure.in	2008-03-09 08:06:18.000000000 -0500
+++ src/gmanedit-0.4.1/configure.in	2008-04-15 01:33:09.703125000 -0500
@@ -17,7 +17,7 @@
 AC_SUBST(GTK_CFLAGS)
 AC_SUBST(GTK_LIBS)
 
-AM_GNU_GETTEXT([external])
+AM_GLIB_GNU_GETTEXT
 dnl Add the languages which your application supports here.
 ALL_LINGUAS="gl es ja ca"
 
